website/content/docs/secrets/aws.mdx

```diff
@@ -47,6 +47,17 @@ Vault policies to have access to the IAM credentials.
 
 Please see the [API documentation](/vault/api-docs/secret/aws#create-static-role) for details on this feature.
 
+### Plugin Workload Identity Federation (WIF)
+The AWS secrets engine supports the Plugin WIF workflow, and has a source of identity called
```

Comment:
This is mostly great! Suggesting some slight rephrasing. Lmk your thoughts.

> The AWS secrets engine supports the Plugin WIF workflow, and has a source of identity called
a plugin identity token. The plugin identity token is a JWT that is internally signed by
Vault's plugin [identity token provider](/vault/api-docs/secret/identity/tokens#read-plugin-identity-well-known-configurations). Given a trust relationship is configured between Vault and AWS via 
[Web Identity Federation](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_providers_oidc.html), the secrets engine can exchange its identity token for temporary STS credentials needed to perform its actions. This enables the secrets engine to operate without needing sensitive IAM security credentials as a part of its configuration.

Reply:
Rephrasing looks good to me! Updated, thanks!